scss/civicrm/contact/_tabs.scss

```diff
@@ -6,15 +6,16 @@
 
   #mainTabContainer {
     background-color: transparent;
+    display: flex;
     padding: 0;
 
     .ui-tabs-nav {
       background: $crm-white;
       border-radius: 0;
       box-shadow: $box-shadow;
       float: left;
-      margin-bottom: -99999px;
-      padding: 0 0 99999px;
+      margin-right: 20px;
```

Comment:
why did you add this right margin?

Reply:
when we do tab setting with page,
All childs will be aligned like table cells without space, so, to give it right space, I added 20px margin on right which is standard space we should have here.

Without margin:
![image](https://user-images.githubusercontent.com/26058635/37821213-10a387da-2ea9-11e8-9c94-a6922277839a.png)